README.md

```diff
@@ -30,9 +30,8 @@ Badge]](./LICENSE)<br>
 [![Codecov Badge]][Codecov] [![Deps.rs Badge]][Deps.rs] [![Discord Badge]][Discord Server]
 [![Matrix Badge]][Matrix]<br>
 
-[Ratatui Website] · [API Docs] · [Examples]<br>
-[Contributing] · [Changelog] · [Breaking Changes]<br>
-[Report a bug] · [Request a Feature] · [Create a Pull Request]
+[Ratatui Website] · [API Docs] · [Examples] · [Changelog] · [Breaking Changes]<br>
+[Contributing] · [Report a bug] · [Request a Feature] · [Create a Pull Request]
 
```

Comment:
Is this normal?

Reply:
Yep. We generate the Readme from lib.rs using cargo-rdme. This was updated and the readme wasn't generated with the new changes. Updating the image updated this automatically.

Reply:
Alright